app/assets/stylesheets/global.sass

```diff
@@ -70,11 +70,10 @@ html, body
     margin-top: 12px
     outline: none
     text-align: center
-    @media screen and (min-width: $screen-md)
+    @media screen and (min-width: $screen-sm)
       display: none
     i
       position: relative
-      top: 5px
```

Comment:
you're of course correct @nbogis, I'm so sorry about this! please ignore my above comment and leave as is 👍